### PR TITLE
Visitor can see list of assignments

### PR DESCRIPTION
**Pull request by FacundoOs**
# [PT](https://www.pivotaltracker.com/story/show/175525242)

## This PR contains: 
- Components for show the assignments and for each card
- Module for the get request
- Add axios
- Add react-navigation
- Add native base for styling

Screen Shot 2020-11-03 at 11 55 41

### CODE REVIEW

File: src/components/AssignmentCard.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { View, StyleSheet, FlatList } from "react-native";
+import {
+  Container,
+  Header,
+  Content,
+  Card,
+  CardItem,
+  Thumbnail,
```
Comment (Saralundkvist86) [inline src/components/AssignmentCard.js:9]: Just remove imported stylingobj we dont use

**Review by mauroavellaneda — APPROVED** [review]

**Review by ctl-lab — APPROVED** [review]

File: App.js [hunk 2]
```diff
@@ -1,18 +1,29 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import AssignmentsIndex from "./src/components/AssignmentsIndex";
+
+const Stack = createStackNavigator(); 
+
+const myOptions = {
```
Comment (oliverochman) [inline App.js]: put the styling under the export default

File: index.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
 import React from "react";
 import { registerRootComponent } from "expo";
 import App from "./App";
-registerRootComponent(App);
\ No newline at end of file
+import axios from "axios";
+
+axios.defaults.baseURL = "http://localhost:3000/api/v1";
```
Comment (oliverochman) [inline index.js]: add production url

Comment (oliverochman) [reply index.js]: mind the namespace

File: src/components/AssignmentCard.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          <Body>
+            <Text>{assignment.title}</Text>
+            <Text note>$ {assignment.budget}</Text>
+          </Body>
+        </Left>
+      </CardItem>
+      <CardItem footer bordered style={styles.container}>
+        <Left>
+          <Text note style={styles.container2}>
+            Points:{" "}
```
Comment (oliverochman) [inline src/components/AssignmentCard.js]: !

File: src/components/AssignmentCard.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      </CardItem>
+    </Card>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: "#4A6572",
+  },
+  container2: {
```
Comment (oliverochman) [inline src/components/AssignmentCard.js]: fix naming of this class

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Pull request merged**